q2_feature_table/_summarize/_visualizer.py

```diff
@@ -356,12 +361,43 @@ def _frequencies(table, axis):
     return pd.Series(data=table.sum(axis=axis), index=table.ids(axis=axis))
 
 
-def _frequency_summary(table, axis='sample'):
+def is_integer(x):
+    if isinstance(x, int):
+        return True
+    elif isinstance(x, float):
+        return x.is_integer()
+    else:
+        return False
+
+
+def _frequency_summary(table: biom.Table, axis='sample'):
     frequencies = _frequencies(table, axis=axis)
 
-    summary = pd.Series([frequencies.min(), frequencies.quantile(0.25),
-                         frequencies.median(), frequencies.quantile(0.75),
-                         frequencies.max(), frequencies.mean()],
+    nonzeros = table.matrix_data.data
+
+    _round = True
+
+    for item in nonzeros:
+        if not is_integer(item):
+            _round = False
+            break
+
+    first = frequencies.quantile(0.25)
```

Comment:
@Clockwork-Rat, a lot of this logic seems left over from the handling of relative frequencies - is that right? I think this can mostly be stripped at this point.

Reply:
I will remove this, @ebolyen had told me it was necessary to leave in even with dropping support for relative frequency, but I don't believe we should be checking this anymore.